packages/core/src/constants.ts

```diff
@@ -96,3 +96,5 @@ export const PluginsPriority = [
 
 export const PLUGIN_STATUS_OK = 'ok'
 export const PLUGIN_STATUS_FAIL = 'fail'
+
+export const kReplyHeaders = Symbol.for('microfleet:replyHeaders')
```

Comment:
To achieve independence from transport packages, it is better to declare the symbol in the core package, and use `Symbol.for (...)` in the other packages as needed.